src/handlers/assign.rs

```diff
@@ -564,6 +571,33 @@ pub(super) async fn handle_command(
                 }
                 name
             }
+            AssignCommand::Reroll => {
+                // We need to compute the PR diff here, but the IssuesEvent created from a
+                // comment webhook doesn't contain the required `base` and `head` fields.
+                // So we have to load the information about the pull request from the GitHub API
+                // explicitly.
+                let pr = ctx
+                    .github
+                    .pull_request(issue.repository(), issue.number)
+                    .await?;
```

Comment:
Could you add some `.context("...")` to all the newly added `?`, so we can more easily detect where the error came from.

Reply:
Done!